contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.4.0--5.0.0.sql

```diff
@@ -206,7 +229,7 @@ BEGIN
     WHERE rolname IN ('securityadmin', 'dbcreator');
         
     IF existing_server_roles IS NOT NULL THEN
-            RAISE EXCEPTION 'The following role(s) already exist(s): %', existing_server_roles;   
+            RAISE WARNING 'The following fixed server role(s) already exist(s): %', existing_server_roles;   
```

Comment:
we should check for is_fixed_role for server roles as well.

Reply:
We only support fixed server role currently not user defined server role. So no need for that.

Reply:
What if user created a login named 'securityadmin' or uses this name to create role from postgres endpoint ?

Reply:
I think we need to use bbf catalog in that case.